client/app/hearings/components/assignHearings/AssignHearingsFields.jsx

```diff
@@ -119,3 +119,20 @@ SuggestedHearingLocation.propTypes = {
     facilityType: PropTypes.string
   })
 };
+
+export const HearingRequestType = ({ hearingRequestType, isFormallyTravel }) => {
+  if (!hearingRequestType) {
+    return null;
+  }
+
+  return (
+    <React.Fragment>
+      {isFormallyTravel ? <span>{`formally Travel, ${hearingRequestType}`}</span> : <span>{hearingRequestType}</span>}
```

Comment:
```suggestion
      {isFormallyTravel ? <span>{`former Travel, ${hearingRequestType}`}</span> : <span>{hearingRequestType}</span>}
```
[the design](https://www.figma.com/proto/V87TZArfdurCGJiEjQ73ES/Virtual-Hearings?node-id=7050%3A20&scaling=min-zoom) shows this as "former Travel"